engine/results/detection_result.py

```diff
@@ -1,38 +1,44 @@
+"""Module containing the `DetectionResult` class for final results of detection."""
+
 from json import dumps as json_dumps
 
 
 class DetectionResult:
     """
-    Represents the final result of a detection query.
+    Represent the final result of a detection query.
```

Comment:
I'd probably write:

```
Representation of the ...
```

Reply:
It sounds awful, but I wanted to avoid having "of the"/"of a" twice in such a short sentence. `The final result of a detection query.` might work even better. Also, this change was an unfortunate and unintentional consequence of me using regular expressions to fix >500 docstyle violations.